### ISSUE TITLE
Linter not using settings from package.json

### ISSUE THREAD

**Issue by Nevon** [Flet/SublimeLinter-contrib-standard#19]
I'm using some testing libraries that declare some global variables like `expect`, `describe`, etc. I've declared those globals in my package JSON as such:

```
  "standard": {
    "ignore": [
      "**/build/",
      "/node_modules/"
    ],
    "globals": [
      "describe",
      "it",
      "expect",
      "$expect",
      "sinon",
      "beforeEach",
      "afterEach",
      "before",
      "after"
    ]
  }
```

If I run `standard` manually, it does not complain about these globals. However, the linter in sublime is still showing errors. So it looks like the linter plugin is not respecting the configuration in the package.json.

**Comment by Nevon** [comment]
Nevermind. I'm not sure why it's working now, since I didn't change anything, but perhaps I had to restart or something. Now it's not complaining anymore.
